utils/http/errors.py

```diff
@@ -0,0 +1,12 @@
+class ResponseError(Exception):
+    """Generic ResponseError exception
```

Comment:
Let's make the docstring better and more accurate after this next round of changes.

Reply:
I added a small extra line for `status_code`. If you want any other changes I can add them too.